pdns/recursordist/docs/changelog/pre-4.0.rst

```diff
@@ -2064,8 +2064,7 @@ Improvements
 Recursor version 3.0.1
 ----------------------
 
-Released 25th of April 2006,
-`download <http://www.powerdns.com/en/downloads.aspx>`__.
```

Comment:
This page is gone, so I've removed it. No one should be trying to download pdns 3.*.

Habbie suggests removing the entire document, and I agree, but it's beyond the scope of this PR.